WiserTaskScheduler/WiserTaskScheduler/Core/Services/ParentUpdateService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using GeeksCoreLibrary.Core.DependencyInjection.Interfaces;
+using GeeksCoreLibrary.Core.Models;
+using GeeksCoreLibrary.Modules.Databases.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using WiserTaskScheduler.Core.Enums;
+using WiserTaskScheduler.Core.Interfaces;
+using WiserTaskScheduler.Core.Models;
+using WiserTaskScheduler.Core.Models.ParentsUpdate;
+
+namespace WiserTaskScheduler.Core.Services
+{
+    /// <summary>
+    /// A service to manage all WTS configurations that are provided by Wiser.
```

Comment:
Please update the description since this is from the MainService interface. What exactly will the ParentUpdateService do?

Reply:
done